File: MultiRubroProducts/MultiRubroProducts/DbSet/Provider.cs
namespace MultiRubroProducts.DbSet
{
    public class Provider:BaseEntity
    {
        public Provider()
        {
            Products = new HashSet<Product>();
        }

        public string Name { get; set; } = string.Empty;
        public string Address { get;set; }= string.Empty;
        public string Cellphone { get; set; } = string.Empty;
        public virtual ICollection<Product> Products { get; set;}
    }
}

File: MultiRubroProyectProducts/src/main/java/com/example/multirubroproyectproducts/models/Provider.java
package com.example.multirubroproyectproducts.models;

import lombok.*;
import lombok.experimental.FieldDefaults;

import java.util.List;
import java.util.UUID;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Setter
@Getter
@Builder
@FieldDefaults(level = AccessLevel.PRIVATE)
public class Provider {
    UUID id;
    String name;
    String address;
    String cellphone;
    List<Product> products;
}
